Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -436,11 +464,402 @@
             "itemKey": "$gen_heatmap_microsoft.documentdb/databaseaccounts-Requests-TotalRequestUnits_5",
             "sortOrder": 2
           }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "selectedTab",
+        "comparison": "isEqualTo",
+        "value": "Overview"
+      },
+      "showPin": true,
+      "name": "cosmosDB account metrics"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "# Overview section ends here\r\n <br />  <br />"
```

Comment:
do these text blocks need to be full heading 1?  does it actually need the br's in the text too?

Reply:
Just to separate the different views I added br tags. It just looks more editable after the br tags.

Reply:
fixed